docs/en/sql-reference/aggregate-functions/reference/exponentialmovingaverage.md

```diff
@@ -0,0 +1,70 @@
+---
+toc_priority: 108
+---
+
+## exponentialMovingAverage {#exponential-moving-average}
+
+An aggregate function that calculates the exponential moving average of values for the determined time. 
+
+**Syntax**
+
+```sql
+exponentialMovingAverage(x)(value, timestamp)
+```
+
+Each `value` corresponds to the determinate `timestamp`. The half-decay period is the time interval `x` during which the previous values are taken into account. The function returns a weighted average: the older the time point, the less weight the corresponding value is considered to be.
```

Comment:
I think `half-life` is more clear than `half-decay`

Reply:
Fixed
![image](https://user-images.githubusercontent.com/92211164/142471746-205f10ca-bcca-4bab-81a6-5654953eac13.png)